src/UserSettingsStore.js

```diff
@@ -112,4 +112,14 @@ module.exports = {
             append: true,  // We always append for email pushers since we don't want to stop other accounts notifying to the same email address
         });
     },
+
+    isFeatureEnabled: function(feature: string): boolean {
+        feature = feature.match(/\w+/g).join('_').toLowerCase();
```

Comment:
Yes, that's how they're intended to be used -- with underscored strings (which remain the same after this magic.)